src/Cms/App/CmsSkinConfig.php

```diff
@@ -10,74 +10,69 @@ class CmsSkinConfig
 
     /**
      * Nazwa skóry
-     * @var string
      */
-    private $_name;
+    private string $name;
+
+    /**
+     * Klucz
+     */
+    private string $key;
 
     /**
      * Dostępne szablony
-     * @var array
      */
-    private $_templates = [];
+    private array $templates = [];
 
     /**
      * Ustawia nazwę
-     * @param string $name
-     * @return CmsSkinConfig
      */
-    public function setName($name)
+    public function setName(string $name): CmsSkinConfig
```

Comment:
w różnych klasach na setterach mamy raz `self` a raz pełną nazwę klasy - można by to kiedyś ujednolicić